frontend/eslint.config.mjs

```diff
@@ -0,0 +1,15 @@
+import { ignoreBuildArtifacts } from "@maykinmedia/eslint-config";
+import recommended from "@maykinmedia/eslint-config/recommended";
+
+const config = [
+  ignoreBuildArtifacts(["build", "storybook-static"]),
+  ...recommended,
+  {
+    name: "project/react-hooks/disable-exhaustive-deps",
+    rules: {
+      "react-hooks/exhaustive-deps": "off",
```

Comment:
@svenvandescheur you can easily turn this into `warn` or `error` to have a go at it :wink: